electroncash/commands.py

```diff
@@ -395,8 +396,16 @@ def serialize(self, jsontx):
         keypairs = {}
         inputs = jsontx.get('inputs')
         outputs = jsontx.get('outputs')
-        locktime = jsontx.get('locktime', 0)
-        locktime = jsontx.get('lockTime', locktime)
+        for txin in inputs:
+            token_data = txin.get('token_data')
+            if isinstance(token_data, dict):
+                txin['token_data'] = token.OutputData(
+                    id=bytes.fromhex(token_data['id'])[::-1],
+                    bitfield=token_data['bitfield'],
+                    amount=token_data['amount'],
+                    commitment=bytes.fromhex(token_data['commitment']) if token_data['commitment'] else b''
```

Comment:
Old C++ programmer nit.. it bothers me to do the same lookup twice in a line of code.. I'd do it this way:

```suggestion
                    commitment=bytes.fromhex(token_data['commitment'] or '')
```